gradle.properties

```diff
@@ -26,11 +26,11 @@ developmentEnvironmentUserName = "Developer"
 # The string's content will be replaced with your mods version when compiled. You should use this to specify your mod's
 # version in @Mod([...], version = VERSION, [...])
 # Leave these properties empty to skip individual token replacements
-replaceGradleTokenInFile = Lib.java
```

Comment:
There is no reason to change this, it was working before and fulfilled all requirements.

Reply:
I can rename it back to `Lib.java` though it doesn't change much either way - I was going for consistency across mods.

Reply:
Just revert https://github.com/GTNewHorizons/BetterLoadingScreen/pull/7/commits/9fb604127455126c74feae6cd034f17a2ccd33a6 and https://github.com/GTNewHorizons/BetterLoadingScreen/pull/7/commits/89f6d38f4c6e3f17003db43ced3b6047ee58afd7

Reply:
(And add the version field as I described in the other comment thread)